crates/ruff_cli/src/commands/format.rs

```diff
@@ -347,6 +399,23 @@ impl Display for FormatSummary<'_> {
                     formatted += 1;
                 }
                 FormatResult::Unchanged => unchanged += 1,
+                FormatResult::Diff {
+                    path,
+                    unformatted,
+                    formatted: formatted_source,
```

Comment:
Why rename here, but leave `unformatted` as-is? Can we either leave them both, or rename them both, for consistency?